### ISSUE TITLE
Permission denied: '/data/config'

### ISSUE THREAD

**Issue by danielborges93** [Chorss/docker-pgAdmin4#2]
Using the command `docker run -d -p 5050:5050 -v /home/user/data:/data chorss/docker-pgadmin4` I'm getting this error:

```
Traceback (most recent call last):
  File "./usr/local/lib/python2.7/site-packages/pgadmin4/pgAdmin4.py", line 55, in <module>
    exec(open(file_quote(setupfile), 'r').read())
  File "<string>", line 46, in <module>
  File "/usr/local/lib/python2.7/site-packages/pgadmin4/pgadmin/setup/data_directory.py", line 23, in create_app_data_directory
    _create_directory_if_not_exists(os.path.dirname(config.SQLITE_PATH))
  File "/usr/local/lib/python2.7/site-packages/pgadmin4/pgadmin/setup/data_directory.py", line 15, in _create_directory_if_not_exists
    os.mkdir(_path)
OSError: [Errno 13] Permission denied: '/data/config'
```

**Comment by sumpfgottheit** [comment]
you can try my container, which is based on chorss/docker-pgAdmin4 and resolves this issue: https://hub.docker.com/r/sumpfgottheit/docker-pgadmin4/. Just use the environment variables UID and GID and set them to your uid and gid. the files created in /data will be owned by this uid/gid.

**Comment by Chorss** [comment]
Fixed in version 2.0

**Comment by abhijitgujar86** [comment]
I am having the same issue while starting the pgadmin4 docker from chorss/docker-pgadmin4 in tried sudo still no use any idea what is causing it ?
